docs/content/guides/dagster/using-environment-variables-and-secrets.mdx

```diff
@@ -0,0 +1,224 @@
+---
+title: Using environment variables and secrets | Dagster Docs
+description: Learn how to securely define and access environment variables in your Dagster code.
+---
+
+# Using environment variables and secrets
+
+<EnvironmentVariablesIntro />
+
+---
+
+## Declaring environment variables
+
+How environment variables are declared depends on how your Dagster project is deployed.
+
+### Dagster Cloud
+
+Environment variables can be set a variety of ways in Dagster Cloud. Refer to the [Dagster Cloud environment variables guide](/dagster-cloud/developing-testing/environment-variables-and-secrets) for more info.
+
+### Dagster Open Source
+
+Setting environment variables in Dagster Open Source depends on where Dagster is deployed. Refer to the deployment guide for your platform for more info:
+
+<ArticleList>
+  <ArticleListItem
+    title="Amazon Web Services EC2 / ECS"
+    href="/deployment/guides/aws#secrets-management-in-ecs"
+  ></ArticleListItem>
+  <ArticleListItem
+    title="Docker"
+    href="/deployment/guides/docker"
+  ></ArticleListItem>
+  <ArticleListItem
+    title="Kubernetes"
+    href="/deployment/guides/kubernetes/deploying-with-helm#configure-your-user-deployment"
+  ></ArticleListItem>
+</ArticleList>
+
+---
+
+## Accessing environment variables
+
+In this section, we'll demonstrate how to access environment variables once they've been declared. There are two ways to do this:
+
+- [In Python code](#in-python-code), which isn't specific to Dagster
```

Comment:
something about "isn't specific to Dagster" reads weird to me in docs, maybe "like in other Python applications" or leave out entirely?

Reply:
I'm gonna opt to just leave it out, I think you're right that it's weird